README.md

```diff
@@ -4,6 +4,8 @@
 
 > A command line game built in ruby
 
+![screenshot](/assets/app_screenshot.png)
+
```

Comment:
- [ ] **OPTIONAL** It would look better if you added some left margin for your screenshot, the text seems to be a bit cut